test/markup/ebnf/terminators.txt

```diff
@@ -0,0 +1,3 @@
+full-stop = "." .
+semicolon = ";" ;
+end-test = "end" ; (* ending production to test semicolon *)
```

Comment:
Is this comment supposed to mean something if I know EBNF?  I find it indecipherable?

Reply:
The "bug" that gets exhibited when using an incorrect terminator (like a `.` before this patch) is that the name of the next rule (the _production_) doesn't get highlighted. So the final production is necessary to test that semicolons still work as a terminator.

Reply:
Ah, makes perfect sense. :-)